### ISSUE TITLE
UI Inspector doesn't work with LayoutView instead of application.region as suggested with docs

### ISSUE THREAD

**Issue by Palid** [marionettejs/marionette.inspector#297]
Example website: https://www.mynameflow.com/
UI inspector won't work when website's written with AMD. It works for Data, Radio and Activity, though.

**Comment by jasonLaster** [comment]
It works :)  My guess, is that your Application doesn't have regions.

**Comment by Palid** [comment]
@jasonLaster I wrote it. And I can assure you it does have regions. 

Tell me what you need to debug the problem, I'd love to help. ;)

**Comment by jasonLaster** [comment]
@Palid step 2 of the FAQ should help you: https://github.com/marionettejs/marionette.inspector/blob/master/docs/faq.md

**Comment by umeboshi2** [comment]
I could not get this to work either.  I tried to attach the layout view as ```app.layout```, but it didn't work.  I have been following the deprecation warnings concerning the app object, and I decided to use Wreqr to provide access to my region manager, instead of attaching objects to the application.  To try and get inspector to work again, I tried the advice of attaching the layout view to the app, but it would not work.
